spel/scripts/amigen7-build.sh

```diff
@@ -266,9 +268,65 @@ function ComposeChrootCliString {
     then
         CLIOPT_CUSTOMREPO=(-r "${AMIGENREPOSRC}" -b "${AMIGENREPOS}")
     fi
+}
+
+declare -a DISKSETUPCMD
```

Comment:
To go with the other suggestion, change this to `DISKSETUPARGS`...
```suggestion
declare -a DISKSETUPARGS
```